### PR TITLE
Update and Revise README

### PR DESCRIPTION
**Pull request by jmarolf**
- cleanup language discussion section
- have the API section point to public docuementation
- re-write contributing section
- small update to questions section

### CODE REVIEW

**Review by mikadumont — APPROVED** [review]

**Comment by jmarolf** [comment]
CC: @jaredpar @jinujoseph and @vatsalyaagrawal to make sure there are not objections before merging this.

File: README.md [hunk 1]
```diff
@@ -4,60 +4,55 @@
...
+- [Tutorial: Write your first analyzer and code fix](https://docs.microsoft.com/dotnet/csharp/roslyn-sdk/tutorials/how-to-write-csharp-analyzer-code-fix)
+- Useful Tools
+  - [Syntax Visualizer Tool](https://docs.microsoft.com/dotnet/csharp/roslyn-sdk/syntax-visualizer)
+  - [Syntax Quoter Tool](http://roslynquoter.azurewebsites.net)
+  - Browse the source with the [enhanced source view](http://sourceroslyn.io/)
 
-Want to start developing in C# and Visual Basic? Download [Visual Studio 2019](https://www.visualstudio.com/downloads/), which has the latest features built-in. There are 
-also [prebuilt Azure VM images](https://azuremarketplace.microsoft.com/en-us/marketplace/apps/category/compute?search=visual%20studio%202019) available with 
-Visual Studio 2019 already installed.
+## Contribute!
```
Comment (jaredpar) [inline README.md:27]: This section no longer links to our pages on how to setup a Windows or Linux environment. This is really important to having a successful clone and build experience. Need to find a way to include it here.

File: README.md [hunk 2]
```diff
@@ -4,60 +4,55 @@
...
-- https://github.com/dotnet/csharplang for C# specific issues
-- https://github.com/dotnet/vblang for VB-specific features
-- https://github.com/dotnet/csharplang for features that affect both languages
+If you want to suggest a new feature for the C# or Visual Basic languages go here:
+- [dotnet/csharplang](https://github.com/dotnet/csharplang) for C# specific issues
+- [dotnet/vblang](https://github.com/dotnet/vblang) for VB-specific features
+- [dotnet/csharplang](https://github.com/dotnet/csharplang) for features that affect both languages
 
-Discussion about the transition of language design to the new repos is at https://github.com/dotnet/roslyn/issues/18002.
+### Getting started with the Roslyn APIs
```
Comment (jaredpar) [inline README.md:14]: I would put this much lower. In general I would have the README basically increase in contributor complexity as they scroll top to bottom. Dealing with our APIs is far more complex than building the repo. 

Basically optimize for new contributors. Existing contributors probably don't need this info or at least know where to go searching for it.

Comment (jmarolf) [reply README.md:14]: yeah, I kept this in its original location but I agree this should be moved (or even removed)

**Review by jaredpar — APPROVED** [review]

**Pull request merged**